src/core/lib/iomgr/tcp_windows.cc

```diff
@@ -422,16 +539,25 @@ static grpc_endpoint_vtable vtable = {win_read,
                                       win_get_peer,
                                       win_get_fd};
 
+#define MAX_CHUNK_SIZE 32 * 1024 * 1024
+#define MAX_SLICE_COUNT 512
```

Comment:
At least the `MAX_SLICE_COUNT` should also be configurable, but not sure if there is any concern in adding more such (possibly Windows-specific) arguments.